src/topology/builder.rs

```diff
@@ -287,9 +288,10 @@ pub async fn build_pieces(
                                 %error,
                                 component_kind = "sink",
                                 component_type = typetag,
-                                %component_id,
+                                component_id = ?component_key.id(),
+                                component_scope = ?component_key.scope(),
                                 // maintained for compatibility
-                                component_name = %component_id,
+                                component_name = ?component_key.id(),
```

Comment:
I think this is my fault because I told you wrong last time, but `?` == `Debug` and `%` == `Display`, so I think we want `%` here and similar places 😢

Reply:
I fixed these 😉